dependency-analyser/pom.xml

```diff
@@ -80,11 +80,11 @@
         </plugins>
     </build>
     <dependencies>
-        <dependency>
+        <!--<dependency>
```

Comment:
1. Remove lines instead of commenting
2. Update copyright year since you are modifying this file.

Reply:
Took care of these.